django/test/selenium.py

```diff
@@ -0,0 +1,60 @@
+import os
+import types
+from unittest import skipIf
+
+from six import add_metaclass
+
+from django.test import tag
+from django.utils.module_loading import import_string
+from django.utils.six import (
+    get_function_closure, get_function_code, get_function_defaults,
+    get_function_globals,
+)
+
+
+class SeleniumMetaClass(type):
+
+    def __new__(cls, name, bases, attrs):
+        """
+        Dynamically inject new methods for running tests in different browsers
+        when multiple browser specs are provided (e.g. --selenium=firefox,chrome).
+        """
+        browsers = os.environ.get('DJANGO_SELENIUM_SPECS', '').split(',')
+        if browsers[0]:
+            for key in list(attrs.keys()):
+                if isinstance(attrs[key], types.FunctionType) and key.startswith('test'):
+                    method = attrs[key]
+                    for index, spec in enumerate(browsers):
+                        # Create new methods for browsers other than first one.
```

Comment:
I think creating a method for each browsers would make more sense as the method name makes it clear on which of the specified browser failed.